src/twisted/mail/pop3.py

```diff
@@ -945,11 +957,13 @@ def ebMessages(err):
                 if i < 1:
                     raise ValueError()
             except ValueError:
-                self.failResponse("Invalid message-number: %r" % (i,))
+                if not isinstance(i, bytes):
```

Comment:
Isn't this what `intToBytes` is for?

Reply:
Good point.

Reply:
`intToBytes` doesn't quite work well here, if the value is a letter for instance or some random class for instance.